inc/meta-box-html.php

```diff
@@ -578,17 +596,18 @@ class="tagdelbutton -<?php echo esc_attr( $term->name ) ?>"><?php
 						?></span><?php
 					} else {
 						$date = strtotime( $term->name );
-						?><span><a id="<?php echo esc_attr( $taxonomy ) ?>-check-num-<?php echo esc_attr( $i ) ?>"
-								  class="tagdelbutton -<?php echo esc_attr( $term->name ) ?>"><?php
+						?><span><a id="<?php echo esc_attr( $field['taxonomy'] ) ?>-check-num-<?php echo esc_attr( $i ) ?>"
+								  class="tagdelbutton &<?php echo esc_attr( $term->name ) ?>"><?php
```

Comment:
Is this why you changed the `split` argument in the JS? The character doesn't seem to be showing up properly. Also, does it need to be updated on L592 above?